pdfminer/pdfdevice.py

```diff
@@ -169,17 +168,17 @@ def end_page(self, page):
     def begin_tag(self, tag, props=None):
         s = ''
         if isinstance(props, dict):
-            s = ''.join(' %s="%s"' % (utils.enc(k), utils.enc(str(v))) for (k, v)
-                        in sorted(props.iteritems()))
-        out_s = '<%s%s>' % (utils.enc(tag.name), s)
+            s = ''.join(' %s="%s"' % (k, str(v)) for (k, v)
+                        in sorted(props.items()))
+        out_s = '<%s%s>' % (tag.name, s)
         self.outfp.write(utils.make_compat_bytes(out_s))
         self._stack.append(tag)
         return
 
     def end_tag(self):
         assert self._stack, str(self.pageno)
         tag = self._stack.pop(-1)
-        out_s = '</%s>' % utils.enc(tag.name)
+        out_s = '</%s>' % tag.name
```

Comment:
Why do you remove `utils.enc`? Sanitizing html values seems like a proper thing to do.